.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: true
+contact_links:
+- name: Ask a question
+  url: https://community.cesium.com/c/cesiumjs/5
+  about: Please use the community forum for general questions about using CesiumJS. We actively monitor it and love seeing what people are working on
```

Comment:
My 
```suggestion
  about: CesiumJS is a big project, so we use GitHub for feature requests and bug tracking exclusively. Please use the community forum for general questions. The Cesium team and community actively monitor it and love seeing what people are working on!
```

Just a suggestion. I pulled some of this from my saved reply text.

Reply:
@ggetz I had trimmed it down a little bit to try and avoid having too big a chunk of text. Can definitely change to this if you want. I do agree it provides a bit more info. Up to you?

![2024-02-02_13-51](https://github.com/CesiumGS/cesium/assets/8007967/980f5afa-0802-460c-acd5-9c3ed3fa341e)

![2024-02-02_13-51_1](https://github.com/CesiumGS/cesium/assets/8007967/cc90d1c8-d06d-43c2-b89d-ec3c274f8cad)

Reply:
I personally think the extra context helps. But I'm not going to bikeshed.